articles/2015-05-15.md

```diff
@@ -0,0 +1,27 @@
+# io.js Semaine du 15 Mai 2015
+io.js rejoint la Fondation Node, événements de la communauté et plus encore
```

Comment:
Pas sûr, l'originale dit `io.js move to the Node Foundation, other community updates and events`

Habituellement dans les autres news on le traduit par `mises à jour de la communauté et événements`

Par contre il vrai que les `événements` en question semblent être ceux de la communauté. Les autres traductions sont peut être à côté de la plaque sur ce point là.